src/firewall/mod.rs

```diff
@@ -45,13 +44,9 @@ enum FirewallImpl {
 /// What firewall implementations does this system support?
 fn get_firewall_impl(driver_name: Option<String>) -> NetavarkResult<FirewallImpl> {
     // It respects "firewalld", "iptables", "nftables", "none".
-
-    // If not requested lookup in NETAVARK_FW env var as well.
-    let driver = driver_name.or_else(|| env::var("NETAVARK_FW").ok());
-
-    if let Some(var) = driver {
-        debug!("Forcibly using firewall driver {}", var);
-        match var.to_lowercase().as_str() {
+    if let Some(driver) = driver_name {
+        debug!("Forcibly using firewall driver {driver}");
+        match driver.to_lowercase().as_str() {
```

Comment:
Can we retain the environment variable here if driver_name is is none?

Reply:
> Can we retain the environment variable here if driver_name is is none?

I'm already getting the value from the environment variable via clap's env attribute, so it should be consistent with the original behavior when no arguments are specified.

Reply:
Clearly I need more coffee

Reply:
> Clearly I need more coffee

But you don't look like you've been up all night or anything.